dova/www/js/controllers.js

```diff
@@ -1,8 +1,13 @@
 angular.module('app.controllers', [])
   
 .controller('signupCtrl', ['$scope', '$stateParams', '$http', '$state', '$rootScope',
+    
 function ($scope, $stateParams, $http, $state, $rootScope) {
+    $scope.offline_debug = false;
```

Comment:
Nice add-on: one thing, I think doing a $scope variable is gonna slow down the app, what about using a js var instead? (If you can think of a way to centralize the debug variable, it will be wonderful)